### PR TITLE
doc minor fixes

### PR DESCRIPTION
**Pull request by isidroas**
- **refactor doc: use "not in" operator**
- **fix doc typo: renamed test_{div,index_of}**

### CODE REVIEW

**Comment by orsinium** [comment]
Thank you!

File: docs/basic/tests.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ contract_for_index_of = deal.chain(
     ),
     # element at this position is the first match
     deal.ensure(
-        lambda items, item, result: not any(el == item for el in items[:result]),
+        lambda items, item, result: item not in items[:result]),
```
Comment (orsinium) [inline docs/basic/tests.md:99]: Tests fail:

https://github.com/life4/deal/actions/runs/11729658216/job/32697214453?pr=136

Could you remove the closing parenthesis, please?

**Comment by orsinium** [comment]
Thanks you! Now it's good :)

**Comment by isidroas** [comment]
This project is awesome 🤟 I am enjoying reading the documentation

**Pull request merged**